overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -2597,7 +2605,7 @@ func (s *deviceMgrRemodelSuite) TestRemodelUC20SwitchKernelBaseGadgetSnapsInstal
 		"label":     expectedLabel,
 		"directory": filepath.Join(boot.InitramfsUbuntuSeedDir, "systems", expectedLabel),
 		// none of the tasks are downloads so they were not tracked
```

Comment:
the comment seems wrong now, also won't it be a problem that we have tasks that are thought to be download but hat are not downloads actually? in relation for example to the create recovery system code?

Reply:
so perhaps it's most convenient to introduce a new edge specifically for remodel to separate tasks that perform actual system state modification from the rest.

Reply:
that might be clearer so I'm not against, but my main original questions was whether this marking things as the branch done confuses the remodel code as is currently?

Reply:
the comment still need changing afaiu